src/main/java/duke/command/FindCommand.java

```diff
@@ -0,0 +1,43 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.TaskList;
+import duke.tasks.Task;
+import duke.tasks.taskType;
+import duke.ui.Ui;
+
+import java.util.ArrayList;
+
+public class FindCommand extends Command{
+    private String keyword;
+
+    public FindCommand(String keyword) {
+        this.keyword = keyword;
+
+    }
+
+    /**
+     * executes the search command on the specified tasklist
+     *
+     * @param taskList tasklist to search
+     * @throws DukeException
+     */
+    @Override
+    public String execute(TaskList taskList) {
+        ArrayList<Task> results = taskList.find(keyword);
+        return Ui.showFindResults(results);
+    }
+
+    /**
+     * returns the type of command
```

Comment:
Conventionally, the description starts with a cap and ends with a fullstop (i.e. `Returns the type of command.`).